views/components/value_content.erb

```diff
@@ -0,0 +1,17 @@
+<%# locals: (content:, opts: nil) %>
+<% opts ||= {} %>
+<% if opts[:copyable] && content %>
+  <%== render("components/copyable_content", locals: { content:, revealable: opts[:revealable] }) %>
+<% elsif opts[:revealable] && content %>
+  <%== render("components/revealable_content", locals: { content: }) %>
```

Comment:
I realize this just moves the logic, but in my work in #2551, I determined that we don't have any components that are revealable but not copyable, so this branch is never taken, and can be removed.